api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerAuthenticationScramSha512.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+
+/**
+ * Configures the Kafka Mirror Maker authentication
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class KafkaMirrorMakerAuthenticationScramSha512 extends KafkaMirrorMakerAuthentication {
+    private static final long serialVersionUID = 1L;
+
+    public static final String TYPE_SCRAM_SHA_512 = "scram-sha-512";
+
+    private String username;
+    private PasswordSecretSource passwordSecret;
+
+    @Description("Must be `" + TYPE_SCRAM_SHA_512 + "`")
+    @Override
+    public String getType() {
+        return TYPE_SCRAM_SHA_512;
+    }
+
+    @Description("Password used for the authentication.")
+    public PasswordSecretSource getPasswordSecret() {
```

Comment:
Description is wrong. It's not the password, it's a reference to a `Secret` which holds a password

Reply:
I recommend "Reference to the `secret` which holds the password"